File: maxNumber.js
const prompt = require('prompt-sync')();
let max;
let number;

number = Number(prompt("Enter number 1:"));
max = number;

for (let i = 2; i <= 10; i++) {
  number = Number(prompt(`Enter number ${i}:`));
  if (number > max) {
    max = number;
  }
}

console.log("Maximum number is:", max);

File: maxNumber.c
#include <stdio.h>

int main() {
    int max, number;

    printf("Enter 10 numbers:\n");

    scanf("%d", &number);
    max = number;

    for (int i = 1; i < 10; i++) {
        scanf("%d", &number);
        if (number > max) {
            max = number;
        }
    }

    printf("Maximum number is: %d\n", max);
    return 0;
}